packages/std/src/init_handle.rs

```diff
@@ -39,6 +41,35 @@ pub enum BankMsg {
 /// those contracts that don't explicitly set a custom message.
 pub enum NoMsg {}
 
+#[cfg(feature = "staking")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum StakingMsg {
+    Delegate {
+        // delegator is automatically set to address of the calling contract
+        validator: HumanAddr,
+        amount: Coin,
+    },
+    Undelegate {
+        // delegator is automatically set to address of the calling contract
+        validator: HumanAddr,
+        amount: Coin,
+    },
+    Withdraw {
+        // delegator is automatically set to address of the calling contract
+        validator: HumanAddr,
+        // this is the "withdraw address", the one that should receive the rewards
+        // if None, then use delegator address
```

Comment:
Can we make such field-specific comments doc comments (`///`) to allows IDEs and JSON schema to utilize them? Note that when rendered there is no resulting line break as long as it is the same paragraph, so a trailing `.` would be good.

Of course, this is not for `// delegator is automatically set to address of the calling contract`, which is not a field description.